src/core/Akka.Remote/Routing/RemoteRouterConfig.cs

```diff
@@ -94,13 +94,9 @@ internal override Routee NewRoutee(Props routeeProps, IActorContext context)
                 .AttachChild(Local.EnrichWithPoolDispatcher(routeeProps, context).WithDeploy(deploy), false, name);
             return new ActorRefRoutee(actorRef);
         }
-
-        // TODO: why internal?
```

Comment:
Old comment asking a question answered by this PR - why was this marked as `internal` in the first place?